test-resources/test_routing.json

```diff
@@ -1229,5 +1229,49 @@
       "23487067": false,
       "23487607": true
     }
+  },
+  {
+    "testName": "68. vehicle:conditional=no https://github.com/osmandapp/OsmAnd/issues/19773 (1)",
+    "params": {
+      "vehicle": "car",
+      "minRouteLength": 140,
```

Comment:
no need to support weird minroutelength... 
We should proper support "expectedResults":  if needed points then points, so it's easier to understand what's the test case
```
"4692930:1": true,
"4692930:8": false
```

Reply:
improved (slightly other way)